ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-master.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: Dockerfile.rhel
```

Comment:
I sent https://github.com/openshift/cluster-api-provider-kubevirt/pull/3 to rename the file to Dockerfile.rhel

Reply:
Thanks I approved it